plugins/module_utils/entries.py

```diff
@@ -0,0 +1,21 @@
+from ansible.module_utils.common.dict_transformations import recursive_diff
```

Comment:
I think you need to add a GPL header here, according to https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#collection-licensing-requirements